packages/framework/src/Model/Complaint/Mail/ComplaintMailFacade.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\Complaint\Mail;
+
+use Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileFacade;
+use Shopsys\FrameworkBundle\Model\Complaint\Complaint;
+use Shopsys\FrameworkBundle\Model\Complaint\Status\ComplaintStatus;
+use Shopsys\FrameworkBundle\Model\Mail\Mailer;
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplate;
+use Shopsys\FrameworkBundle\Model\Mail\MailTemplateFacade;
+
+class ComplaintMailFacade
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Mail\Mailer $mailer
+     * @param \Shopsys\FrameworkBundle\Model\Mail\MailTemplateFacade $mailTemplateFacade
+     * @param \Shopsys\FrameworkBundle\Model\Complaint\Mail\ComplaintMail $complaintMail
+     * @param \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileFacade $uploadedFileFacade
+     */
+    public function __construct(
+        protected readonly Mailer $mailer,
+        protected readonly MailTemplateFacade $mailTemplateFacade,
+        protected readonly ComplaintMail $complaintMail,
+        protected readonly UploadedFileFacade $uploadedFileFacade,
+    ) {
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Complaint\Complaint $complaint
+     */
+    public function sendEmail(Complaint $complaint): void
```

Comment:
shouldn't it be used somewhere?

Reply:
you are right 😅 added, should it be somewhere else as well?